atd-vze/src/views/Crashes/Notes.js

```diff
@@ -46,6 +50,43 @@ const Notes = ({ crashId }) => {
     }).catch(error => console.error(error));
   };
 
+  // function to handle edit button click
+  const handleEditClick = (row) => {
+    setEditRow(row);
+  };
```

Comment:
@roseeichelmann I think that you can test out setting `editedNote` in this function to the note's text value that is available in the `row` object here. This update would make sure that the value of `editedNote` initializes to the the row's current value right when it you click the edit button instead of `""`. Hopefully, that helps out here!

Reply:
This worked! Thank you Mike!